gen/templates/datasource_test.go.tmpl

```diff
@@ -100,7 +100,13 @@ data "aci_{{$.resourceName}}" "test" {
 }
 `
 
+{{- if (eq $.resourceClassName "FvSiteAssociated") }}
```

Comment:
why is this needed? Dont think we should hard code classnames in the template but determine the difference in behaviour